hammr/utils/bundle_utils.py

```diff
@@ -74,11 +74,13 @@ def recursivelyAppendToArchive(bundle, files, parentDir, checkList, archive_file
     #must save the filepath before changing it after archive
     filePathBeforeTar = files["source"]
     if not "tag" in files or ("tag" in files and files["tag"] != "ospkg"):
-        #if parentDir is a no empty path, add os.sep after. Else keep it as ""
-        if parentDir:
+        # if parentDir is a no empty path or already ending with os.sep, add os.sep at the end
```

Comment:
Could be extracted to a proper method to avoid duplicate at line 107, couldn't it?